components/event-gateway/README.md

```diff
@@ -0,0 +1,43 @@
+# Event-Gateway
+
+## Overview
+
+This is the repository for the Kyma Event-Gateway.
+
+## Installation
+
+To install the Event-Gateway, follow these steps:
+
+1. `git clone git@github.com:kyma-project/kyma.git`
+1. `cd kyma/components/event-gateway`
+1. `CGO_ENABLED=0 go build ./cmd/eventgateway`
+
+## Usage
+
+The Event-Gateway has the following parameters:
+- **externalAPIPort** - This port exposes the Event-Gateway API to an external solution. The default port is `8081`.
+- **eventsTargetURL** - A URL to which you proxy the incoming events. The default URL is http://localhost:9000.
```

Comment:
`the incoming Events.`

Reply:
@klaudiagrz this is done as part of another merged pull request.